pyqtgraph/graphicsItems/ImageItem.py

```diff
@@ -52,9 +57,12 @@ def __init__(self, image=None, **kargs):
         self.lut = None
         self.autoDownsample = False
         self._lastDownsample = (1, 1)
-        self._buffer = None
+        self._gpuBuffer = None
```

Comment:
This might be better named primaryBuffer, or gpuOrCpuBufferWhicheverIsAppropriate